config.ru

```diff
@@ -1,4 +1,20 @@
 # This file is used by Rack-based servers to start the application.
 
+if ENV['RAILS_ENV'] == 'production'
+  require 'unicorn/worker_killer'
+
+  max_request_min =  500
+  max_request_max =  600
+
+  # Max requests per worker
+  use Unicorn::WorkerKiller::MaxRequests, max_request_min, max_request_max
+
+  oom_min = (240) * (1024**2)
+  oom_max = (260) * (1024**2)
+
+  # Max memory size (RSS) per worker
+  use Unicorn::WorkerKiller::Oom, oom_min, oom_max
```

Comment:
At some point when we have a better understanding of our server configurations, we could parameterize the request and memory values based on servers.